pom.xml

```diff
@@ -174,6 +175,11 @@
                 <artifactId>java-hamcrest</artifactId>
                 <version>2.0.0.0</version>
             </dependency>
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-api</artifactId>
+                <version>${version.junit-jupiter}</version>
+            </dependency>
```

Comment:
Maybe consider using the JUnit BOM (https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-bom).

Reply:
Nice one; added a commit. Couldn't nicely get rid of `${version.junit-jupiter}`, because the version is still shared with the Surefire dependency declaration.